tests/test_app.rs

```diff
@@ -242,13 +249,13 @@ fn test_params() {
     fn post_badge_1(
         _p: web::Path<KnownResourceBadge>,
         _q: web::Query<BadgeParams>,
-        _f: web::Form<BadgeBody>,
+        _f: web::Form<BadgeForm>,
     ) -> String {
         unimplemented!();
     }
 
     #[api_v2_operation]
-    fn post_badge_2(_p: web::Path<(String, String)>, _b: web::Json<BadgeBody>) -> String {
+    fn post_badge_2(_p: web::Path<(String, String)>, _b: web::Json<BadgeForm>) -> String {
```

Comment:
Can you change this to `BadgeBody`, so that we can actually test what we're trying to do?

Reply:
Cool! sure.